### PR TITLE
chore: update readme, add .env.local, update docker-compose

### PR DESCRIPTION
**Pull request by ReedSoftware**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,5 +6,10 @@ This repository hosts the Alerts User Interface, a frontend application for mana
...
 
-For more instructions and general hmpps project setup guidelines:
-- [Project Setup Instructions](docs/HMPPS_PROJECT_SETUP).
+  * Make a copy of the `.env.local` file and rename it to `.env`
+  * Ensure that `HMPPS_ALERTS_UI_APPINSIGHTS_KEY`, `HMPPS_ALERTS_UI_API_CLIENT_ID`, `HMPPS_ALERTS_UI_API_CLIENT_SECRET`, `HMPPS_ALERTS_UI_SYSTEM_CLIENT_ID`, `HMPPS_ALERTS_UI_SYSTEM_CLIENT_SECRET` and any other system vars prefixed with `$` in your new `.env` file are populated via your `.zprofile` or `.bash_profile` file, or whatever normal method you use to set system variables. If that's not possible, edit these references in your .env file (NOT the .env.local file) and hard-code the secrets/keys in. This file is ignored by git so should never be committed. In order to get the actual values of these, you can put something like this `export HMPPS_ALERTS_UI_API_CLIENT_ID=$(kubectl get secrets/hmpps-alerts-ui --template={{.data.API_CLIENT_ID}} | base64 -d)` in your `.zprofile`, and repeat for other env vars needed. Please see https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/kubectl-config.html for guidance on K8s access.
```
[the remark this answers is not in the document]
Comment (ReedSoftware) [reply README.md:10]: Okay - if creds don't change often/at all, happy to change the example to hardcoded example (with a made up value of course)

File: README.md [hunk 2]
```diff
@@ -6,5 +6,10 @@ This repository hosts the Alerts User Interface, a frontend application for mana
...
+  * Ask David Winchurch or someone else with access to manage user accounts in HMPPS Digital Services, to create you a new user
```
[the remark this answers is not in the document]
Comment (ReedSoftware) [reply README.md:11]: Happy to remove his name and make it generic :)

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (iterative-it) [reply README.md:10]: Yes agree with Bill. It's fine to say that the real values should be retrieved from secrets and reference the guide but no need to have the command itself in the profile

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (iterative-it) [reply README.md:11]: Yeah leave me out of this :)

**Pull request merged**